lte/gateway/deploy/agw_post_install.sh

```diff
@@ -79,3 +79,43 @@ if [ -n "$INFO" ]; then
     echo "INFO:"
     printf "%s" "$INFO"
 fi
+
+echo "- Check for Root Certificate"
+CA=/var/opt/magma/tmp/certs/rootCA.pem
+if [ -d "/var/opt/magma/tmp/certs/" ]; then
+    if [ -f "$CA" ]; then
+        echo "$CA exists"
+    fi
+else
+    echo "Verify Root CA in /var/opt/magma/tmp/certs/"
+    echo "Access Gateway configurations failed"
+fi
+
+echo "- Check for Control Proxy"
+CP=/var/opt/magma/configs/control_proxy.yml
+if [ -d "/var/opt/magma/configs/" ]; then
+    if [ -f "$CP" ]; then
+        echo "$CP exists"
+        echo "- Check control proxy content"
+        cp_content=("cloud_address" "cloud_port" "bootstrap_address" "bootstrap_port" "rootca_cert")
+        for content in "${cp_content[@]}"; do
+            if ! grep -q $content $CP; then
+                echo "Missing $content in control proxy"
```

Comment:
qq, do you think it might be better to also indicate where the service for e.g. cloud_port or cloud_address or fluentd_address is running ?